pkg/cloud/ratelimit.go

```diff
@@ -112,3 +112,108 @@ func (m *MinimumRateLimiter) Accept(ctx context.Context, key *RateLimitKey) erro
 func (m *MinimumRateLimiter) Observe(ctx context.Context, err error, key *RateLimitKey) {
 	m.RateLimiter.Observe(ctx, err, key)
 }
+
+type TickerRateLimiter struct {
+	ticker *time.Ticker
+}
+
+func NewTickerRateLimiter(limit int, interval time.Duration) *TickerRateLimiter {
+	return &TickerRateLimiter{
+		ticker: time.NewTicker(interval / time.Duration(limit)),
+	}
+}
+
+// Accept everything immediately.
+func (t *TickerRateLimiter) Accept(ctx context.Context, rlk *RateLimitKey) error {
+	select {
+	case <-t.ticker.C:
+		return nil
+	case <-ctx.Done():
+		return ctx.Err()
+	}
+}
+
+// Observe does nothing.
```

Comment:
// Observe implements RateLimiter

Reply:
I believe that `Observe does nothing` is a better comment as it doesn't implement anything (only satisfies an interface - I added separate check for that).

Reply:
we can see that this function does nothing because it is empty what we don't know is why this function is empty and I think that it is good to put this in comment otherwise the comment is redundant.